flowauth/backend/flowauth/config.py

```diff
@@ -0,0 +1,60 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import logging
+
+import os
+from pathlib import Path
+from cryptography.fernet import Fernet
+
+
+def getsecret(key: str, default: str) -> str:
+    """
+    Get a value from docker secrets (i.e. read it from a file in
+    /run/secrets), return a default if the file is not there.
+
+    Parameters
+    ----------
+    key: str
+        Name of the secret.
+    default: str
+        Default value to return if the file does not exist
+
+    Returns
+    -------
+    str
+        Value in the file, or default
+    """
+    try:
+        with open(Path("/run/secrets") / key, "r") as fin:
+            return fin.read().strip()
+    except FileNotFoundError:
+        return default
+
+
+def get_config():
+    FLOWAUTH_FERNET_KEY = getsecret(
+        "FLOWAUTH_FERNET_KEY", os.environ["FLOWAUTH_FERNET_KEY"]
```

Comment:
One issue I found when working on #838 was that this invocation will raise an error when using secrets only (even if the secret is defined), because Python will try to access `os.environ["FLOWAUTH_FERNET_KEY"]` _before_ the call to `getsecret()`. I fixed this by pushing the fallback to the environment variable into the helper function (see [here](https://github.com/Flowminder/FlowKit/blob/b26c54c45eeda952e58b2991f222d69802dcc974/flowapi/flowapi/config.py#L15)). Might be worth using a similar pattern here.

Reply:
Ahh yes, I spotted that. It was making me think that the `get_secret` function is a perfect example of something that should live in a common module.